test/e2e/network/netpol/network_policy.go

```diff
@@ -350,6 +350,34 @@ var _ = SIGDescribeCopy("Netpol [LinuxOnly]", func() {
 			ValidateOrFail(k8s, model, &TestCase{FromPort: 81, ToPort: 80, Protocol: v1.ProtocolTCP, Reachability: reachability})
 		})
 
+		ginkgo.It("should enforce policy based on multiple label PodSelectors (OR) [Feature:NetworkPolicy]", func() {
+			nsX, nsY, nsZ, model, k8s := getK8SModel(f)
+			peers := []networkingv1.NetworkPolicyPeer{
```

Comment:
wondering if these peers would be better off defined in the `GetAllowIngressByMultiLabeledPod` function

🤔 not sure

Reply:
Maybe passing a list of dicts `[]map[string]string{ {"pod": "a", "pod": "b"} }`, and handling inside the function
```
func GetAllowIngressByAnyPod(name string, targetLabels map[string]string, peersLabel []map[string]string) *networkingv1.NetworkPolicy {
	policyPeers := []networkingv1.NetworkPolicyPeer{}
	for _, label := range peersLabel {
		policyPeers = append(policyPeers, networkingv1.NetworkPolicyPeer{
			PodSelector: &metav1.LabelSelector{MatchLabels: label},
		})
	}
```